cmake/modules/RootTestDriver.cmake

```diff
@@ -72,10 +72,9 @@ endif()
 
 #---Set environment --------------------------------------------------------------------------------
 if(ENV)
-  string(REPLACE "@" "=" _env ${ENV})
```

Comment:
Why did you need to remove this?

Reply:
For environment variables à la `ASAN_OPTIONS=verbose=1`, this wreaks havoc. With the regex parsing just below, this isn't necessary, any more. The regex parsing will only split at the first `=`, and all others can stay intact.

Reply:
However, since I discovered the undocumented `__asan_default_options` weak symbol magic, this might not be necessary, any more. Unless we decide to use `gcc`, that is, which probably doesn't have the weak symbol magic, which would force us to pass environment variables across everything again.